### PR TITLE
[fontconfig] Add cflags in tools build command

### PR DESCRIPTION
**Pull request by xandox**
add cflags in tools build command

- What does your PR fix? Fixes # fixes needs for PR #15605 

- Which triplets are supported/not supported? same
- Have you updated the CI baseline? no

- Does your PR follow the [maintainer guide](https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/maintainer-guide.md)? yes

### CODE REVIEW

**Review by NancyLi1013 — APPROVED** [review]

**Comment by vicroms** [comment]
@xandox please pull the latest commit and run `vcpkg x-add-version` again

**Comment by NancyLi1013** [comment]
Seems you missed the changes in `ports/fontconfig/CONTROL ` file.

Could you please add it back? @xandox

**Review by NancyLi1013 — APPROVED** [review]

**Comment by NancyLi1013** [comment]
LGTM now, thanks for your PR @xandox.

File: versions/f-/fontconfig.json [hunk 1]
```diff
@@ -1,7 +1,12 @@
 {
   "versions": [
     {
-      "git-tree": "de83a21d912d9b87d8ad069d94dffcca201830a3",
+      "git-tree": "e4934e51612999eea86c66bcc794fa88aedef173",
+      "version-string": "2.13.1",
+      "port-version": 6
+    },
+    {
+      "git-tree": "d9774e3ba72153356e6ba148678304283053feda",
```
Comment (ras0219-msft) [inline versions/f-/fontconfig.json:9]:
```suggestion
      "git-tree": "de83a21d912d9b87d8ad069d94dffcca201830a3",
```
Previous versions must not be modified.

**Review by ras0219-msft — CHANGES_REQUESTED** [review]

**Comment by strega-nil-ms** [comment]
@xandox could you rebase on head? thanks!

**Comment by xandox** [comment]
up

**Comment by strega-nil-ms** [comment]
Thanks @xandox!

**Pull request merged**